hermes/probe/read/read_test.go

```diff
@@ -1,26 +1,11 @@
-// Copyright 2020 Google LLC
```

Comment:
I assume all source files need the licence header. Is that not so?

Reply:
added